suite-native/passphrase/src/passphraseSlice.ts

```diff
@@ -7,8 +7,11 @@ import {
     createDeviceInstanceThunk,
 } from '@suite-common/wallet-core';
 
+import { verifyPassphraseOnEmptyWalletThunk } from './passphraseThunks';
+
 type PassphraseState = {
     error: AuthorizeDeviceError | CreateDeviceInstanceError | null;
+    isVefifyingPassphraseOnEmptyWallet: boolean;
```

Comment:
```suggestion
    isVerifyingPassphraseOnEmptyWallet: boolean;
```

Reply:
Good catch! Couldn't even see the difference when comparing these 2 diffs :D 
https://github.com/trezor/trezor-suite/pull/12876/commits/a3937fd599788239f40cd82fa5f816f3b1f98bf5